cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ConnectorsTest.java

```diff
@@ -150,6 +150,20 @@ public void testMirrorTargetClusterNotSameAsConnectCluster() {
                 "[Connect cluster alias (currently set to target) has to be the same as the target cluster alias third]"));
     }
 
+
+    @Test
+    public void testClusterNotSameButBootstrapUrlSame() {
+        KafkaMirrorMaker2 kmm2 = new KafkaMirrorMaker2Builder(KMM2)
+                .editSpec()
+                .editCluster(1)
+                .withBootstrapServers("source:9092")
+                .endCluster()
+                .withConnectCluster("source")
+                .endSpec()
+                .build();
```

Comment:
You can make the test much easier to read and maintain by actually adding a new alias for it and not just edit one based on number.